src/components/Accordion/Accordion.module.scss

```diff
@@ -1,55 +1,18 @@
 @use '../../assets/styles/theme';
 
-.card-header {
```

Comment:
Why you have removed these bunch of CSS classes? Won't these changes break the design?

Reply:
No. It was unnecessary